articles/cosmos-db/unique-keys.md

```diff
@@ -130,6 +130,20 @@ The following command sample shows how to create a unique index on the firstName
 ```
 db.users.createIndex( { firstName: 1, lastName: 1, email: 1 }, { unique: true } )
 ```
+## Azure Portal sample
```

Comment:
Please change capitalization to "Azure portal sample". We use sentence caps in headings, and Azure portal is capitalized as "Azure portal" per our internal style guidelines.